pom.xml

```diff
@@ -221,6 +237,19 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
```

Comment:
Should this also have `notimestamp` set? Looked like it said so in the docs.

Reply:
I initially thought so too (coincidentally also looked into reproducible builds this weekend), but the [`<notimestamp>` documentation](https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#notimestamp) states:
> **Note**: If the project has the property `project.build.outputTimestamp`, the value will be overwritten to true. This way it is possible to generate reproducible javadoc jars.